server/routes/components.ts

```diff
@@ -24,7 +24,8 @@ export default function routes({ serviceCatalogueService, redisService }: Servic
     const component = await serviceCatalogueService.getComponent({ componentName })
     const dependencies = (await redisService.getAllDependencies()).getDependencies(componentName)
     const { environments } = component
-
+    const prodEnvData = component.environments?.filter(environment => environment.name === 'prod')
```

Comment:
filter using the environment.type

Reply:
environment.name to check environment type is correct